CMakeLists.txt

```diff
@@ -435,6 +436,9 @@ OCV_OPTION(WITH_MFX "Include Intel Media SDK support" OFF
 OCV_OPTION(WITH_GDAL "Include GDAL Support" OFF
   VISIBLE_IF NOT ANDROID AND NOT IOS AND NOT XROS AND NOT WINRT
   VERIFY HAVE_GDAL)
+OCV_OPTION(WITH_QOI "Include QOI Support" OFF
```

Comment:
Could you please add temporary test commit which enables this flag (to run CI tests)?

Reply:
Thank you for you review. I created an independent commit just there.
And I could notice that miss-commit test data and build-warning on Visual Studio.
Thank you very much !